salt/modules/file.py

```diff
@@ -6415,6 +6416,7 @@ def delete_backup(path, backup_id):
 
     return ret
 
+
```

Comment:
Can we pull newline out please? I'm not sure why it was added.

Reply:
same, E305

Is it OK to ignore it?